media.info.yml

```diff
@@ -0,0 +1,8 @@
+name: Media
+description: 'Media module for Drupal 8'
+type: module
+package: Media
+core: 8.x
+dependencies:
+  - media_entity_image
```

Comment:
The media_entity_image module lists media_entity as a dependency but not the modules which are required by media_entity such as the entity module. 
So I assumed I had to only list direct dependencies and other dependencies would be handled automatically. 
I'll fix this and send a new pull request.